misc/GNUmakefile

```diff
@@ -30,12 +30,12 @@ release: export
 
 export:
 	rm -rf $(TEMP)
-	hg archive -X '.hg*' $(TEMP)/$(NGINX)
+	git archive --prefix=$(TEMP)/$(NGINX)/ HEAD | tar x
```

Comment:
- for consistency with existing usage, tar command operands should be specified with hyphens such as: "tar -x"
- for better interoperability, it makes sense to explicitly specify a file operand (instead of "the system-dependent default" as per ISO/IEC 9945-1:1996 (“POSIX.1”). In particular it makes the command work on Solaris where the system default is a tape device ("/dev/rmt").
OTOH, tar operands specified on Solaris with hyphens work in an unexpected way unless the operands are grouped with a single hyphen.
That said, the most consistent (and arguably portable) way would be: "tar -xf -"